{"uuid": "a0673c1a-4806-4e31-b438-df064b6691d8", "children": ["933c8860-a103-46e9-99fa-770bebe8585d"], "befores": [{"name": "log_on_failure", "status": "passed", "start": 1741258533837, "stop": 1741258533837}], "afters": [{"name": "log_on_failure::0", "status": "broken", "statusDetails": {"message": "AttributeError: 'Function' object has no attribute 'rep_call'\n", "trace": "  File \"C:\\Users\\Administrator\\PycharmProjects\\Ecom-app\\.venv\\Lib\\site-packages\\allure_commons\\_allure.py\", line 221, in __call__\n    return self._fixture_function(*args, **kwargs)\n           ~~~~~~~~~~~~~~~~~~~~~~^^^^^^^^^^^^^^^^^\n  File \"C:\\Users\\Administrator\\PycharmProjects\\Ecom-app\\.venv\\Lib\\site-packages\\_pytest\\fixtures.py\", line 907, in _teardown_yield_fixture\n    next(it)\n    ~~~~^^^^\n  File \"C:\\Users\\Administrator\\PycharmProjects\\Ecom-app\\test_cases\\conftest.py\", line 20, in log_on_failure\n    if item.rep_call.failed:\n       ^^^^^^^^^^^^^\n"}, "start": 1741258541074, "stop": 1741258541424}], "start": 1741258533837, "stop": 1741258541424}